_wiki/FAQ/Linux.md

```diff
@@ -14,21 +14,17 @@ Also check out [Linux App Specific FAQ]({% link _wiki/FAQ/LinuxAppSpecific.md %}
 
 If your tablet is connected properly and is supported, but is still not detected, [check logs]({% link _wiki/Documentation/Logs.md %}) for any errors or warnings. If you find any, try finding for a match and its accompanying solution below:
 
-##### UCLogic kernel module is loaded {#hid_uclogic}
+##### Another driver is found {#hid_uclogic}
 
 _Symptoms_
 
 ```
-Another tablet driver found: UC Logic
-```
-
-```
-ArgumentOutOfRangeException: Value range is [0, 15]. (Parameter 'value)
```

Comment:
Is this actually no longer displayed for users? I would have to check with a uclogic tablet to see whether it still displays this, before I can merge this change.

Reply:
The tablet driver detected message is enough to cover this as well. Only time that this message occurs is when `hid_uclogic` is loaded, which makes this redundant.